example/eblob_backend.cpp

```diff
@@ -747,95 +748,206 @@ struct iterated_key_info {
 };
 
 typedef std::function<int (const iterated_key_info &info)> iterator_callback;
+typedef std::function<int (std::shared_ptr<iterated_key_info> info)> iterator_server_send_callback;
 
-static iterator_callback make_iterator_server_send_callback(eblob_backend_config *c, dnet_net_state *st,
-                                                            dnet_cmd *cmd,
-                                                            ioremap::elliptics::dnet_server_send_request &request,
-                                                            uint64_t iterator_id,
-                                                            uint64_t &counter) {
+/*
+ * congestion_control_monitor is used for limiting amount of data being sent to a remote backend.
+ */
+class congestion_control_monitor
+{
+public:
+	/*!
+	 * Constructor: initializes internal state properly.
```

Comment:
What does mean **properly** here? Is there any way to initialize it **improperly**?